src/ngraph/op/experimental/quantized_dot.cpp

```diff
@@ -45,6 +47,8 @@ op::QuantizedDot::QuantizedDot(const shared_ptr<Node>& data,
                           " weights shape ",
                           weights_shape);
 
-    auto output_et = requantize ? (with_relu ? element::u8 : element::i8) : element::f32;
+    auto output_et = is_output_i32
```

Comment:
This nested trinary is a little dense for me.  Prefer that you unpack it into a few lines.